src/components/Pricing.tsx

```diff
@@ -169,21 +46,261 @@ export const Pricing = () => {
+    { title: 'Unlimited apps' },
+    { title: '1 developer' },
+    { title: '5K patch installs/month' },
+    { title: 'Community support' },
+  ];
+  return (
+    <div className="w-[325px] px-1 mb-8 lg:mb-0">
+      <div>
+        <div className="h-full p-6 bg-shorebirdBg3 rounded-3xl">
+          <p className="mb-2 text-xl font-bold font-heading text-white text-left">
+            Hobby
+          </p>
+          <div className="flex justify-start items-end">
+            <div className="text-4xl sm:text-5xl font-bold text-white text-left mt-4 mr-2">
+              $0
+            </div>
+            <div className="text-gray-500">{'/ month'}</div>
+          </div>
+          <p className="mt-4 mb-4 text-gray-500 leading-loose text-left">
+            For small apps and demos.
+          </p>
+          <ul className="mb-2 2xl:mb-6 text-white">
+            {features.map((feature, index) => (
+              <li className="mb-4 flex" key={`hobby-${feature.title}-${index}`}>
+                <CheckArrowIcon />
+                <span className="flex gap-1">{feature.title}</span>
+              </li>
+            ))}
+          </ul>
+          <a
+            target="_blank"
+            href={config.consoleUrl}
+            className="inline-block text-center py-2 px-4 w-full rounded-xl rounded-t-xl shorebird-button-primary font-bold leading-loose mt-8"
+          >
+            Get Started
+          </a>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+function PriceSlider({
+  prices,
+  onChange,
+}: {
+  prices: Price[];
+  onChange: (price: Price) => void;
+}) {
+  const [priceIndex, setPriceIndex] = useState(0);
+  const price = prices[priceIndex];
+  return (
+    <div className="flex w-full flex-col">
+      <div className="flex flex-row items-end">
+        <div className="text-4xl sm:text-5xl font-bold text-white text-left mt-4 mr-2">
+          {price.amount}
+        </div>
+        <div className="text-gray-500 justify-end">{'/ month'}</div>
+      </div>
+      <div className="h-4"></div>
+      <input
+        className="accent-shorebirdPrimary bg-red"
+        type="range"
+        min={0}
+        max={prices.length - 1}
+        value={priceIndex}
+        onChange={(e) => {
+          const index = parseInt(e.target.value);
+          setPriceIndex(index);
+          onChange(prices[index]);
+        }}
+      ></input>
+    </div>
+  );
+}
+
+function TeamPlan() {
+  const prices: Price[] = [
+    {
+      amount: '$20',
+      quantity: '50K',
+    },
+    {
+      amount: '$100',
+      quantity: '300K',
+    },
+    {
+      amount: '$300',
+      quantity: '1M',
+    },
+    {
+      amount: '$700',
+      quantity: '2.5M',
+    },
+    {
+      amount: '$1,250',
+      quantity: '$5M',
+    },
+    {
+      amount: '$2,000',
+      quantity: '10M',
+    },
+  ];
+
+  const features: Feature[] = [
+    { title: 'Unlimited apps' },
+    { title: 'Unlimited developers' },
+    { title: 'Discord & email support' },
```

Comment:
I'd like this to change to private support beyond 5M?

Reply:
sounds good